client/templates/HomeView.ejs

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>IBM Blockchain Platform</title>
+    <style>
+        body {
+            display: flex;
+            padding: 30px;
+        }
+
+        a {
+            text-decoration: none;
+            font-weight: 300;
+        }
+
+        p {
+            margin-top: 6px;
+            margin-bottom: 6px;
+        }
+
+        h2 {
+            font-weight: 100;
+            font-size: 1.6em;
+        }
+
+        h3 {
+            font-weight: 100;
+        }
+
+        /*
+        ##############################################
+        ### Styling for vscode-dark grouped themes ###
+        ##############################################
+        */
+
+        body.vscode-dark .samples-panel {
+            background-color: #282828;
+        }
+
+        body.vscode-dark .repository-name {
+            color: #B1B1B1;
+        }
+
+        /*
+        ##############################################
+        ### Styling for vscode-light grouped themes ###
+        ##############################################
+        */
+
+        body.vscode-light {
+            color: #000000;
+        }
+
+        body.vscode-light .samples-panel {
+            background-color: #F3F3F3;
+        }
+
+        body.vscode-dark .repository-name {
+            color: #F3F3F3;
+        }
+    </style>
+</head>
+
+<body>
+    <div style="flex-shrink:1;max-width:64px;">
```

Comment:
why do you have lots of styles in the html why not put them in the style bit or is it possible to have a separate style sheet